package.json

```
@@ -0,0 +1,41 @@
+{
+  "name": "go-scraper",
+  "version": "1.0.0",
+  "description": "Project repository template to set up all public projects at [Nimble](https://nimblehq.co/)",
+  "scripts": {
+    "dev": "webpack --watch --config conf/webpack.dev.js",
+    "build": "NODE_ENV=production webpack --config conf/webpack.prod.js"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hoangmirs/go-scraper.git"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hoangmirs/go-scraper/issues"
+  },
+  "homepage": "https://github.com/hoangmirs/go-scraper#readme",
+  "devDependencies": {
+    "@babel/core": "^7.12.9",
```

Prefer using exact package version instead to make sure not even patch releases are not introduced.

```suggestion
    "@babel/core": "7.12.9",
```

You can add this dot file `.npmrc` to make sure this behaviour is followed each time a new dependency is added.

```
save-exact=true
```